#point in world coordinates
geometry_msgs/PointStamped world_point
---
#point in image coordinates
geometry_msgs/Point image_point
